ci-operator/templates/openshift-azure/cluster-launch-e2e-azure-conformance.yaml

```diff
@@ -0,0 +1,236 @@
+# This template is being used for testing OpenShift on Azure. 
+# At the moment it cannot be reused to launch cluster for Origin tests
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: LOCAL_IMAGE_BIN
+  required: true
+- name: IMAGE_ORIGINTEST
+  required: true
+- name: IMAGE_SYNC
+  required: true
+- name: NAMESPACE
+  required: true
+- name: CLUSTER_TYPE
+  value: "azure"
+  required: true
+
+objects:
+
+# We want the cluster to be able to access these images
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-image-puller
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: system:image-puller
+  subjects:
+  - kind: SystemGroup
+    name: system:unauthenticated
+
+# The e2e pod spins up a cluster, runs e2e tests, and then cleans up the cluster.
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: ${JOB_NAME_SAFE}
+    namespace: ${NAMESPACE}
+    annotations:
+      # we want to gather the teardown logs no matter what
+      ci-operator.openshift.io/wait-for-container-artifacts: teardown
+  spec:
+    restartPolicy: Never
+    activeDeadlineSeconds: 10800
+    terminationGracePeriodSeconds: 600
+    volumes:
+    - name: artifacts
+      emptyDir: {}
+    - name: shared-tmp
+      emptyDir: {}
+    - name: openshift-tmp
+      emptyDir: {}
+    - name: cluster-secrets-azure
+      secret:
+        secretName: e2e-azure-secret
+
+    containers:
+
+    # Once admin.kubeconfig exists, executes shared tests
+    - name: test
+      image: registry.svc.ci.openshift.org/azure/origin-test:v3.10
```

Comment:
Why not use the upstream openshift/origin-tests image?

Reply:
need to check it if it does what we need to do. Never used it.

Reply:
The image does not container uncompiled tests and code :/ does not work for us. it does not even has go in it.

Reply:
It doesn't need to have go, does it? It should contain a binary with all the Origin conformance tests.

Reply:
hmmm. found them `./ginkgo extended.test`  need to check if it contains all what we need. would be good to swap